[2014-05-15 18:22:08] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429
Stack trace:
#0 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException))
#1 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#2 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(530): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#3 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(506): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#4 /Users/programmer/sites/ace/index.php(50): Illuminate\Foundation\Application->run()
#5 {main} [] []
[2014-05-15 18:22:08] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429
Stack trace:
#0 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException))
#1 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#2 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(530): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#3 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(506): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#4 /Users/programmer/sites/ace/index.php(50): Illuminate\Foundation\Application->run()
#5 {main} [] []
[2014-05-15 18:22:20] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429
Stack trace:
#0 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException))
#1 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#2 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(530): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#3 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(506): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#4 /Users/programmer/sites/ace/index.php(50): Illuminate\Foundation\Application->run()
#5 {main} [] []
[2014-05-15 18:22:21] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429
Stack trace:
#0 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException))
#1 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#2 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(530): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#3 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(506): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#4 /Users/programmer/sites/ace/index.php(50): Illuminate\Foundation\Application->run()
#5 {main} [] []
[2014-05-15 18:22:31] log.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'Class 'Gy' not found' in /Users/programmer/sites/ace/app/modules/widget/controllers/WidgetController.php:217
Stack trace:
#0 [internal function]: Illuminate\Exception\Handler->handleShutdown()
#1 {main} [] []
[2014-05-15 18:43:28] log.ERROR: exception 'Illuminate\Database\Eloquent\ModelNotFoundException' in /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Database/Eloquent/Builder.php:121
Stack trace:
#0 /Users/programmer/sites/ace/app/routes.php(358): Illuminate\Database\Eloquent\Builder->firstOrFail()
#1 [internal function]: {closure}('solutions')
#2 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array)
#3 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable()
#4 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request))
#5 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(530): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#6 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(506): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#7 /Users/programmer/sites/ace/index.php(50): Illuminate\Foundation\Application->run()
#8 {main} [] []
